components/UI/SitesIcon/Icons.tsx

```diff
@@ -0,0 +1,140 @@
+import Avatar from "@mui/material/Avatar";
+import Stack from "@mui/material/Stack";
+import Popover from "@mui/material/Popover";
+import Box from "@mui/material/Box";
+import Typography from "@mui/material/Typography";
+
+import React from "react";
+
+const SitesIcon = () => {
+  const [anchorEl, setAnchorEl] = React.useState<null | HTMLElement>(null);
+  const [isOpen, setIsOpen] = React.useState<any>(null);
+  const handlePopoverOpen = (Popover: string) => {
+    const element = document.getElementById("sitesIcon");
+    setAnchorEl(element);
+    setIsOpen(Popover);
+  };
+
+  const handlePopoverClose = () => {
+    setAnchorEl(null);
+    setIsOpen(null);
+  };
+
+  return (
+    <Stack
+      id="sitesIcon"
+      direction="row"
+      spacing={2}
+      sx={{
+        background: "rgba(220, 20, 60, 0.5)",
+        padding: "10px",
+        borderRadius: "10px",
+        position: "absolute",
+        top: "10px",
+        right: "10px",
+        zIndex: 1000,
+      }}
+    >
+      <Box>
+        <a
```

Comment:
Can we use next/link for routing ? Did you try this before ? <Link href="" />

Reply:
fixed